################################################################################
# PkgEval set-up: 2020-10-29T05:10:25.848
#

Julia Version 1.6.0-DEV.1363
Commit f727f4ba82 (2020-10-28 23:50 UTC)
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: AMD EPYC 7502 32-Core Processor
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.0 (ORCJIT, znver2)
Environment:
  JULIA_DEPOT_PATH = ::/usr/local/share/julia
  JULIA_MAX_NUM_PRECOMPILE_FILES = 9223372036854775807



################################################################################
# Installation: 2020-10-29T05:10:27.317
#

   Resolving package versions...
   Installed Pio3d ────────── v0.0.2
   Installed VersionParsing ─ v1.2.0
   Installed Conda ────────── v1.4.1
   Installed Parsers ──────── v1.0.11
   Installed MacroTools ───── v0.5.6
   Installed PyCall ───────── v1.92.1
   Installed JSON ─────────── v0.21.1
Updating `~/.julia/environments/v1.6/Project.toml`
  [276a7f50] + Pio3d v0.0.2
Updating `~/.julia/environments/v1.6/Manifest.toml`
  [8f4d0f93] + Conda v1.4.1
  [682c06a0] + JSON v0.21.1
  [1914dd2f] + MacroTools v0.5.6
  [69de0a69] + Parsers v1.0.11
  [276a7f50] + Pio3d v0.0.2
  [438e738f] + PyCall v1.92.1
  [81def892] + VersionParsing v1.2.0
  [2a0f44e3] + Base64
  [ade2ca70] + Dates
  [8f399da3] + Libdl
  [37e2e46d] + LinearAlgebra
  [d6f4376e] + Markdown
  [a63ad114] + Mmap
  [de0858da] + Printf
  [9a3f8284] + Random
  [9e88b42a] + Serialization
  [4ec0a83e] + Unicode
    Building Conda ─→ `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/7a58bb32ce5d85f8bf7559aa7c2842f9aecf52fc/build.log`
    Building PyCall → `~/.julia/scratchspaces/44cfe95a-1eb2-52ea-b672-e2afdf69b78f/b6dff5fa725eff4f775f472acd86756d6e31fb02/build.log`


################################################################################
# Testing: 2020-10-29T05:11:24.086
#


     Testing Pio3d
Status `/tmp/jl_Z4SID3/Project.toml`
  [276a7f50] Pio3d v0.0.2
  [438e738f] PyCall v1.92.1
  [8dfed614] Test
Status `/tmp/jl_Z4SID3/Manifest.toml`
  [8f4d0f93] Conda v1.4.1
  [682c06a0] JSON v0.21.1
  [1914dd2f] MacroTools v0.5.6
  [69de0a69] Parsers v1.0.11
  [276a7f50] Pio3d v0.0.2
  [438e738f] PyCall v1.92.1
  [81def892] VersionParsing v1.2.0
  [2a0f44e3] Base64
  [ade2ca70] Dates
  [b77e0a4c] InteractiveUtils
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [de0858da] Printf
  [9a3f8284] Random
  [9e88b42a] Serialization
  [8dfed614] Test
  [4ec0a83e] Unicode
     Testing Running tests...
Pio3d.jl: Error During Test at /home/pkgeval/.julia/packages/Pio3d/qsitX/test/runtests.jl:6
  Test threw exception
  Expression: "nrn4" in Pio3d.get_labels()
  PyError (PyImport_ImportModule
  
  The Python package io3d.datasets could not be imported by pyimport. Usually this means
  that you did not install io3d.datasets in the Python version being used by PyCall.
  
  PyCall is currently configured to use the Julia-specific Python distribution
  installed by the Conda.jl package.  To install the io3d.datasets module, you can
  use `pyimport_conda("io3d.datasets", PKG)`, where PKG is the Anaconda
  package the contains the module io3d.datasets, or alternatively you can use the
  Conda package directly (via `using Conda` followed by `Conda.add` etcetera).
  
  Alternatively, if you want to use a different Python distribution on your
  system, such as a system-wide Python (as opposed to the Julia-specific Python),
  you can re-configure PyCall with that Python.   As explained in the PyCall
  documentation, set ENV["PYTHON"] to the path/name of the python executable
  you want to use, run Pkg.build("PyCall"), and re-launch Julia.
  
  ) <class 'ModuleNotFoundError'>
  ModuleNotFoundError("No module named 'io3d'")
  
  Stacktrace:
   [1] pyimport(name::String)
     @ PyCall ~/.julia/packages/PyCall/BcTLp/src/PyCall.jl:547
   [2] get_labels()
     @ Pio3d ~/.julia/packages/Pio3d/qsitX/src/Pio3d.jl:52
   [3] macro expansion
     @ ~/.julia/packages/Pio3d/qsitX/test/runtests.jl:6 [inlined]
   [4] macro expansion
     @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
   [5] top-level scope
     @ ~/.julia/packages/Pio3d/qsitX/test/runtests.jl:6
Pio3d.jl: Error During Test at /home/pkgeval/.julia/packages/Pio3d/qsitX/test/runtests.jl:4
  Got exception outside of a @test
  PyError (PyImport_ImportModule
  
  The Python package io3d.datasets could not be imported by pyimport. Usually this means
  that you did not install io3d.datasets in the Python version being used by PyCall.
  
  PyCall is currently configured to use the Julia-specific Python distribution
  installed by the Conda.jl package.  To install the io3d.datasets module, you can
  use `pyimport_conda("io3d.datasets", PKG)`, where PKG is the Anaconda
  package the contains the module io3d.datasets, or alternatively you can use the
  Conda package directly (via `using Conda` followed by `Conda.add` etcetera).
  
  Alternatively, if you want to use a different Python distribution on your
  system, such as a system-wide Python (as opposed to the Julia-specific Python),
  you can re-configure PyCall with that Python.   As explained in the PyCall
  documentation, set ENV["PYTHON"] to the path/name of the python executable
  you want to use, run Pkg.build("PyCall"), and re-launch Julia.
  
  ) <class 'ModuleNotFoundError'>
  ModuleNotFoundError("No module named 'io3d'")
  
  Stacktrace:
    [1] pyimport(name::String)
      @ PyCall ~/.julia/packages/PyCall/BcTLp/src/PyCall.jl:547
    [2] datasets_download(label::String)
      @ Pio3d ~/.julia/packages/Pio3d/qsitX/src/Pio3d.jl:33
    [3] macro expansion
      @ ~/.julia/packages/Pio3d/qsitX/test/runtests.jl:7 [inlined]
    [4] macro expansion
      @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Test/src/Test.jl:1144 [inlined]
    [5] top-level scope
      @ ~/.julia/packages/Pio3d/qsitX/test/runtests.jl:6
    [6] include(fname::String)
      @ Base.MainInclude ./client.jl:444
    [7] top-level scope
      @ none:6
    [8] eval(m::Module, e::Any)
      @ Core ./boot.jl:360
    [9] exec_options(opts::Base.JLOptions)
      @ Base ./client.jl:261
   [10] _start()
      @ Base ./client.jl:485
Test Summary: | Error  Total
Pio3d.jl      |     2      2
ERROR: LoadError: Some tests did not pass: 0 passed, 0 failed, 2 errored, 0 broken.
in expression starting at /home/pkgeval/.julia/packages/Pio3d/qsitX/test/runtests.jl:4

PkgEval failed: Package Pio3d errored during testing
Stacktrace:
  [1] pkgerror(msg::String)
    @ Pkg.Types /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/Types.jl:52
  [2] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, julia_args::Cmd, test_args::Cmd, test_fn::Nothing)
    @ Pkg.Operations /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/Operations.jl:1651
  [3] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec}; coverage::Bool, test_fn::Nothing, julia_args::Cmd, test_args::Cmd, kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:334
  [4] test(ctx::Pkg.Types.Context, pkgs::Vector{Pkg.Types.PackageSpec})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:321
  [5] #test#62
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:71 [inlined]
  [6] test
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:70 [inlined]
  [7] #test#61
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:68 [inlined]
  [8] test
    @ /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:68 [inlined]
  [9] test(pkg::String; kwargs::Base.Iterators.Pairs{Union{}, Union{}, Tuple{}, NamedTuple{(), Tuple{}}})
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:67
 [10] test(pkg::String)
    @ Pkg.API /workspace/srcdir/usr/share/julia/stdlib/v1.6/Pkg/src/API.jl:67
 [11] top-level scope
    @ none:44
 [12] eval
    @ ./boot.jl:360 [inlined]
 [13] eval(x::Expr)
    @ Base.MainInclude ./client.jl:446
 [14] top-level scope
    @ none:1
 [15] eval(m::Module, e::Any)
    @ Core ./boot.jl:360
 [16] exec_options(opts::Base.JLOptions)
    @ Base ./client.jl:261
 [17] _start()
    @ Base ./client.jl:485


################################################################################
# PkgEval teardown: 2020-10-29T05:11:46.081
CPU usage: 73.11s (66.18s user, 7.01s kernel)

Network usage:
- eth0: 252.757 MiB received, 885.414 KiB sent

Raw statistics: {"blkio_stats":{"io_queue_recursive":[],"io_serviced_recursive":[{"major":259,"value":2,"op":"Read","minor":0},{"major":259,"value":15,"op":"Write","minor":0},{"major":259,"value":2,"op":"Sync","minor":0},{"major":259,"value":15,"op":"Async","minor":0},{"major":259,"value":17,"op":"Total","minor":0},{"major":253,"value":2,"op":"Read","minor":2},{"major":253,"value":15,"op":"Write","minor":2},{"major":253,"value":2,"op":"Sync","minor":2},{"major":253,"value":15,"op":"Async","minor":2},{"major":253,"value":17,"op":"Total","minor":2}],"io_time_recursive":[],"sectors_recursive":[],"io_service_bytes_recursive":[{"major":259,"value":8192,"op":"Read","minor":0},{"major":259,"value":61440,"op":"Write","minor":0},{"major":259,"value":8192,"op":"Sync","minor":0},{"major":259,"value":61440,"op":"Async","minor":0},{"major":259,"value":69632,"op":"Total","minor":0},{"major":253,"value":8192,"op":"Read","minor":2},{"major":253,"value":61440,"op":"Write","minor":2},{"major":253,"value":8192,"op":"Sync","minor":2},{"major":253,"value":61440,"op":"Async","minor":2},{"major":253,"value":69632,"op":"Total","minor":2}],"io_wait_time_recursive":[],"io_merged_recursive":[],"io_service_time_recursive":[]},"memory_stats":{"usage":1812877312,"max_usage":2348802048,"stats":{"hierarchical_memory_limit":9223372036854771712,"pgmajfault":0,"total_rss":200040448,"inactive_file":8192,"inactive_anon":1137291264,"total_pgfault":1481432,"total_cache":1580503040,"total_rss_huge":0,"total_active_anon":643186688,"total_pgmajfault":0,"pgpgin":1416991,"rss_huge":0,"mapped_file":0,"total_inactive_file":8192,"dirty":0,"total_unevictable":0,"total_dirty":0,"writeback":0,"pgpgout":982288,"hierarchical_memsw_limit":0,"cache":1580503040,"active_file":16384,"pgfault":1481432,"total_mapped_file":0,"total_pgpgout":982288,"total_writeback":0,"rss":200040448,"unevictable":0,"total_inactive_anon":1137291264,"total_pgpgin":1416991,"active_anon":643186688,"total_active_file":16384},"limit":540972998656},"name":"/Pio3d-V3K5R2DO","networks":{"eth0":{"rx_bytes":265034680,"rx_packets":24737,"tx_packets":15999,"tx_bytes":906664,"tx_dropped":0,"rx_dropped":2,"tx_errors":0,"rx_errors":0}},"id":"dd9fbc24840f103af54618ee6a7dac6be522758fe057b631aeddaca84c13884b","preread":"0001-01-01T00:00:00Z","storage_stats":{},"precpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"cpu_usage":{"usage_in_usermode":0,"usage_in_kernelmode":0,"total_usage":0}},"pids_stats":{"current":38},"num_procs":0,"cpu_stats":{"throttling_data":{"throttled_periods":0,"throttled_time":0,"periods":0},"system_cpu_usage":102137535820000000,"online_cpus":128,"cpu_usage":{"usage_in_usermode":66180000000,"usage_in_kernelmode":7010000000,"total_usage":73109062139,"percpu_usage":[0,0,0,0,0,0,0,0,0,0,0,0,0,0,951251,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3300,0,73108107588,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]}},"read":"2020-10-29T05:11:46.105366013Z"}